scripts/get_version_labels.py

```diff
@@ -0,0 +1,76 @@
+"""
+Extract version labels from HathiTrust volume pages.
+"""
+import sys
+import os.path
+import re
+import time
+import datetime
+import requests
+
+
+def get_version_label(htid):
+    """
+    Extract the HathiTrust "version label" from a record's catalog page.
+    Returns the corresponding timestamp, returns None if the HTTP request fails.
+    """
+    re_pattern = r'HT.params.versionLabel = "([^"]+)";'
```

Comment:
I think this is worth a brief comment to explain what we're looking for. Here's my attempt, feel free to edit.
```suggestion
    # a script block in HT record page includes a number of HT.params including version timestamp
    re_pattern = r'HT.params.versionLabel = "([^"]+)";'
```